guestbook/README.md

```diff
@@ -4,704 +4,303 @@
 > https://kubernetes.io/docs/tutorials/stateless-application/guestbook/.
 > Otherwise some of the URLs will not work properly.
 
-## Guestbook Example
+## Creating a Guestbook
 <!-- EXCLUDE_FROM_DOCS END -->
 
-This example shows how to build a simple, multi-tier web application using Kubernetes and [Docker](https://www.docker.com/).
+{% capture overview %}
+This tutorial shows you how to build and deploy a simple, multi-tier web application using Kubernetes and [Docker](https://www.docker.com/). This example consists of the following components:
 
-**Table of Contents**
-<!-- BEGIN MUNGE: GENERATED_TOC -->
+* A single-instance [Redis](https://redis.io/) master to store guestbook entries 
+* Multiple replicated Redis instances to serve reads 
+* Multiple web frontend instances
 
-  - [Guestbook Example](#guestbook-example)
-    - [Prerequisites](#prerequisites)
-    - [Quick Start](#quick-start)
-    - [Step One: Start up the redis master](#step-one-start-up-the-redis-master)
-      - [Define a Deployment](#define-a-deployment)
-      - [Define a Service](#define-a-service)
-      - [Create a Service](#create-a-service)
-      - [Finding a Service](#finding-a-service)
-        - [Environment variables](#environment-variables)
-        - [DNS service](#dns-service)
-      - [Create a Deployment](#create-a-deployment)
-      - [Optional Interlude](#optional-interlude)
-    - [Step Two: Start up the redis slave](#step-two-start-up-the-redis-slave)
-    - [Step Three: Start up the guestbook frontend](#step-three-start-up-the-guestbook-frontend)
-      - [Using 'type: LoadBalancer' for the frontend service (cloud-provider-specific)](#using-type-loadbalancer-for-the-frontend-service-cloud-provider-specific)
-    - [Step Four: Cleanup](#step-four-cleanup)
-    - [Troubleshooting](#troubleshooting)
-    - [Appendix: Accessing the guestbook site externally](#appendix-accessing-the-guestbook-site-externally)
-      - [Google Compute Engine External Load Balancer Specifics](#google-compute-engine-external-load-balancer-specifics)
+{% endcapture %}
 
-<!-- END MUNGE: GENERATED_TOC -->
+{% capture objectives %}
+* Start up a Redis master.
+* Start up Redis slaves.
+* Start up the guestbook frontend.
+* Expose and view the Frontend Service.
+* Clean up.
+{% endcapture %}
 
-The example consists of:
+{% capture prerequisites %}
 
-- A web frontend
-- A [redis](http://redis.io/) master (for storage), and a replicated set of redis 'slaves'.
+{% include task-tutorial-prereqs.md %}
+Download the following configuration files:
+1. [redis-master-deployment.yaml](https://kubernetes.io/docs/tutorials/docs/tutorials/stateless-application/redis-master-deployment.yaml)
```

Comment:
add a space below the 1. otherwise they all render on the same line (not as a list). also enumerating them as 1. 2. 3. is not important, you can just call them all 1. 1. 1., it will count the same.

markdown rule of thumb: when in doubt, add spaces.